et/duplicateCaseField.ts

```diff
@@ -83,14 +83,18 @@ export function getObjectsReferencedByCaseFields(config: ConfigSheets, caseField
   const refScrubbed = config.Scrubbed.filter(o => refCaseFields.find(x => x.FieldTypeParameter === o.ID))
   const refAuthCaseField = config.AuthorisationCaseField.filter(o => refCaseFields.find(x => x.ID === o.CaseFieldID))
   const refAuthCaseEvent = config.AuthorisationCaseEvent.filter(o => refCaseEvents.find(x => x.ID === o.CaseEventID))
+  const refComplexType = config.ComplexTypes.filter(o => refCaseFields.find(x => x.ID === o.ID))
```

Comment:
Needs to be `x => x.FieldTypeParameter === o.ID || x.FieldType === o.ID` for collections of complex types